### PR TITLE
fix: do some clean up before retry to install the gpu driver.

### PR DESCRIPTION
**Pull request by andyliuliming**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by CecileRobertMichon** [comment]
@andyliuliming did the same code merge in master already?

File: parts/k8s/cloud-init/artifacts/cse_config.sh [hunk 1]
```diff
@@ -472,7 +491,8 @@ configGPUDrivers() {
     retrycmd_if_failure 120 5 25 pkill -SIGHUP dockerd || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
     mkdir -p $GPU_DEST/lib64 $GPU_DEST/overlay-workdir
     retrycmd_if_failure 120 5 25 mount -t overlay -o lowerdir=/usr/lib/x86_64-linux-gnu,upperdir=${GPU_DEST}/lib64,workdir=${GPU_DEST}/overlay-workdir none /usr/lib/x86_64-linux-gnu || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
-    retrycmd_if_failure 3 1 600 sh $GPU_DEST/nvidia-drivers-$GPU_DV --silent --accept-license --no-drm --dkms --utility-prefix="${GPU_DEST}" --opengl-prefix="${GPU_DEST}" || exit $ERR_GPU_DRIVERS_START_FAIL
+    export -f installGPUDriversRun
```
Comment (xuto2) [inline parts/k8s/cloud-init/artifacts/cse_config.sh:494]: do we have to export?

**Comment by xuto2** [comment]
let's figure out a way for testing and not make changes directly in aks-release branch.

File: parts/k8s/cloud-init/artifacts/cse_config.sh (hunk 1 above)
Comment (andyliuliming) [reply parts/k8s/cloud-init/artifacts/cse_config.sh:494]: yes, this is run as a bash sub-command by the retry entrance, 
we need to export it.

**Comment by andyliuliming** [comment]
> let's test this out in RP private branch first.

deployed one standalone environment to verify, succeeds, and running the scale up/down stress testing now.

**Comment by andyliuliming** [comment]
> @andyliuliming did the same code merge in master already?

not yet, 
here's one PR targeting to master too:
https://github.com/Azure/aks-engine/pull/3018

**Pull request merged**